source/blog/2020-05-04-this-month-in-solidus-april-2020.html.markdown

```diff
@@ -0,0 +1,122 @@
+---
+title: "This Month in Solidus: April 2020"
+date: 2020-05-04
+tags: Solidus
+author: Alessandro Desantis
+---
+
+Hello folks, and welcome to our first edition of This Month in Solidus!
+
+As you may guess from the name, this is a monthly blog post we'll be publishing with news from the
+entire Solidus ecosystem. The goal is to keep you in the loop about what's happening and make sure
+you take advantage of all the latest opportunities Solidus and its community provide you.
+
+Without further ado, here's what happened in the Solidus world in April 2020!
+
+### Business
```

Comment:
I wouldn't call it business, I think it should be something like `Onboarding experience` or `Community experience`. What do you think?